skyrim-platform/src/platform_se/skyrim_platform/EventsApi.cpp

```diff
@@ -459,6 +459,16 @@ void EventsApi::IpcSend(const char* systemName, const uint8_t* data,
   SendEvent("ipcMessage", { JsValue::Undefined(), ipcMessageEvent });
 }
 
+void EventsApi::SendConsoleMsgEvent(char* msg_)
+{
+  std::string msg(msg_);
+  SkyrimPlatform::GetSingleton().AddUpdateTask([=] {
```

Comment:
Please change to `AddTickTask` to work in the main menu
Or tell me why it isn't possible